src/App.css

```diff
@@ -0,0 +1,164 @@
+@font-face {
```

Comment:
You don't need to have all style under App.css, because with React you will split your page in to smaller reusable components, and whey will have specific style